src/woo_publications/publications/api/filters.py

```diff
@@ -59,6 +66,11 @@ class DocumentFilterSet(FilterSet):
         field_name="creatiedatum",
         lookup_expr="lt",
     )
+    creatiedatum_tot_en_met = filters.DateTimeFilter(
```

Comment:
the `DateTimeFilter` tripped me here because I had in mind that the database field is just a `DateField`, and after checking, it is!

So the `creatiedatum_vanaf`, `creatiedatum_tot_en_met` and `creatiedatum_tot` filters should be `filters.DateFilter` rather than `DateTimeFilter`.

All the other fields are indeed datetime fields.